test_data/thipe-ayeye/templates/thipe-ayeye-about-template.html

```diff
@@ -1,14 +1,19 @@
 <html lang="en">
   <head>
     {# Facebook sharing image (Open Graph) #}
-    <meta property="og:url" content="https://ayeye-thipe-akerte.bird-apps.com/ro-crate-preview.html" />
-    <meta property="og:type" content="website" />
-    <meta property="og:title" content="Ayeye Thipe Akerte" />
-    <meta property="og:description" content="Arrernte stories about birds" />
-    <meta property="og:image" content="http://ayeye-thipe-akerte.bird-apps.com/files/images/og-img.jpg" />
-    <meta property="og:image:secure_url" content="https://ayeye-thipe-akerte.bird-apps.com/files/images/og-img.jpg" />
+    {% set domain = data.pages[data.entryPoint].domain %}
+
+    {% if domain %}
+    <meta property="og:url" content="https://{{ domain }}/ro-crate-preview.html" />
```

Comment:
The path here should be the path to the generated page. See comment in the subobj template for a hint